ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -346,11 +345,15 @@ const plusLink = (dataTestId: string, path: string, disable?: boolean) => {
   </Clickable></TooltipTrigger>;
 };
 
-const BoxHeader = ({text= '', header =  '', subHeader = '', style= {}, ...props}) => {
+const StepNumber = ({step, style = {}}) => {
```

Comment:
Opportunistic refactor not required for this fix.  It's a separate commit, if you want to distinguish.